### PR TITLE
Update gau2gridConfig.cmake.in

### PR DESCRIPTION
**Pull request by loriab**
untested, but this should do the trick for #9

### CODE REVIEW

**Comment by loriab** [comment]
actually, that's wrong. just a sec.

File: cmake/gau2gridConfig.cmake.in [hunk 1]
```diff
@@ -49,6 +51,12 @@ set (_valid_components
     shared
 )
 
+if(${SPHERICAL_ORDER} STREQUAL "cca")
```
Comment (dgasmith) [inline cmake/gau2gridConfig.cmake.in]: Will "CCA" match as well?

Comment (loriab) [reply cmake/gau2gridConfig.cmake.in]: no. cmake is rotten at cases. `OR` if you want to be generous to the user. I've done all components in the psi eco to lowercase, so I'd prefer "OR" to outright uppercase.

Comment (loriab) [reply cmake/gau2gridConfig.cmake.in]: not w/o writing some checking code. latest commit to main CM constrains this. I haven't looked for where `SPHERICAL_ORDER` is actually used.

Comment (loriab) [reply cmake/gau2gridConfig.cmake.in]: looks like you weren't using mixed or uppercase in code anyways. so should be ok.

**Comment by loriab** [comment]
After merge, I should build psi against this to check components are working. Then it'd be good to mint a 1.0.2 so that downstream can reliably do `find_package(gau2grid REQUIRED COMPONENTS cca)`.

But g2g as-is, I think.

**Comment by dgasmith** [comment]
Ok, thanks. I might add in more cartesian orderings and mint a v1.1. It is a fairly significant bump.

**Pull request merged**